system/services/siteTree/SiteTreeService.cfc

```diff
@@ -509,6 +509,7 @@ component {
 				, savedFilters       = savedFilters
 				, orderBy            = "sort_order"
 				, allowDraftVersions = args.allowDrafts
+				, useCache           = false
```

Comment:
I'd rather use cache here if we can. The cache timeout is 20 minutes, so we are talking about being accurate to 20 mins for the embargo/expiry date. Given that the navigation is invoked on every page, it makes sense to cache it by default.